policyengine_us/parameters/gov/states/hi/tax/income/credits/lihrtc/age_threshold.yaml

```diff
@@ -0,0 +1,11 @@
+description: Hawaii allows filers at or above this age threshold to receive the income tax credit for low-income household renters.
+values:
+  2022-01-01: 65
```

Comment:
need 2021 values

Reply:
for all yaml files - you can find them in the 2021 tax form - if it is the same as the 2022 then we set the start date to 2021-01-01